ROMFS/px4fmu_common/init.d/rc.sensors

```diff
@@ -409,6 +409,10 @@ then
 	lsm303d start
 fi
 
+if ms5525_airspeed start
```

Comment:
Should there be some if/else logic here with meas_airspeed start and ms5525_airspeed start?